docs/installation/getting-started/quick-start.md

```diff
@@ -4,318 +4,1720 @@
+								'@sugar', '@sweet', '@topping', '@wafer'
+							],
+							minimumCharacters: 1
+						}
+					]
+				},
+				placeholder: 'Type or paste your content here!',
+				table: {
+					contentToolbar: [
+						'tableColumn', 'tableRow', 'mergeTableCells', 'tableProperties', 'tableCellProperties', 'toggleTableCaption'
+					]
+				},
+				wproofreader: {
+				// PROVIDE CORRECT VALUE HERE:
+					serviceId: 'service ID',
+					lang: 'auto',
+					srcUrl: 'https://svc.webspellchecker.net/spellcheck31/wscbundle/wscbundle.js'
+				},
+				pagination: {
+					// A4
+					pageWidth: '21cm',
+					pageHeight: '29.7cm',
+
+					pageMargins: {
+						top: '20mm',
+						bottom: '20mm',
+						right: '12mm',
+						left: '12mm'
+					}
+				},
+				licenseKey: '', // PROVIDE CORRECT VALUE.
+				collaboration: {
+					channelId: 'super-build'
+				},
+				presenceList: {
+					container: document.querySelector( '.presence' )
+				},
+				sidebar: {
+					container: document.querySelector( '.sidebar-container' )
+				},
+				revisionHistory: {
+					showRevisionViewerCallback: config => {
+						const editorContainer = document.querySelector( '#editor-container' );
+						const viewerContainer = document.querySelector( '#revision-viewer-container' );
+						const viewerElement = document.querySelector( '#revision-viewer-editor' );
+
+						config.revisionHistory.viewerSidebarContainer = document.querySelector( '#revision-viewer-sidebar' );
+
+						return CKEDITOR.DecoupledEditor.create( viewerElement, config ).then( viewerEditor => {
+							viewerContainer.style.display = 'flex';
+							editorContainer.style.display = 'none';
+
+							const toolbarContainer = document.querySelector( '.editor-toolbar' );
+							toolbarContainer.innerHTML = '';
+							toolbarContainer.appendChild( viewerEditor.ui.view.toolbar.element );
+
+							return viewerEditor;
+						} );
+					},
+					closeRevisionViewerCallback: viewerEditor => {
+						const editorContainer = document.querySelector( '#editor-container' );
+						const viewerContainer = document.querySelector( '#revision-viewer-container' );
+
+						viewerContainer.style.display = 'none';
+						editorContainer.style.display = '';
+
+						return viewerEditor.destroy().then( () => {
+							const toolbarContainer = document.querySelector( '.editor-toolbar' );
+							toolbarContainer.innerHTML = '';
+							toolbarContainer.appendChild( window.editor.ui.view.toolbar.element );
+						} );
+					}
+			},
 	} )
 	.then( editor => {
-		console.log( 'Editor was initialized', editor );
+		const toolbarContainer = document.querySelector( '.editor-toolbar' );
+
+		toolbarContainer.appendChild( editor.ui.view.toolbar.element );
 	} )
 	.catch( error => {
-		console.error( error.stack );
+		console.log( error );
 	} );
 ```
 
-You can now run webpack to build the application. To do that, call the `webpack` executable:
```

Comment:
As above, the unnecessary huge amount of code and repetition of most of it just behind, I think we might delete this and refer to the main example.